apps/web/src/pages/TrainingOpportunities/TrainingOpportunityPage.tsx

```diff
@@ -0,0 +1,283 @@
+  id: string;
+}
+
+interface TrainingOpportunityProps {
+  query: FragmentType<typeof TrainingOpportunityPage_Fragment>;
+}
+
+const TrainingOpportunityPage = ({ query }: TrainingOpportunityProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const trainingOpportunity = getFragment(
+    TrainingOpportunityPage_Fragment,
+    query,
+  );
+
+  const crumbs = useBreadcrumbs({
+    crumbs: [
+      {
+        label: intl.formatMessage(pageTitles.itTrainingFund),
+        url: paths.itTrainingFund(),
+      },
+      {
+        label: intl.formatMessage(pageTitles.instructorLedTraining),
+        url: paths.instructorLedTraining(),
+      },
+      {
+        label: trainingOpportunity.title?.localized,
+        url: paths.instructorLedTrainingOpportunity(trainingOpportunity.id),
+      },
+    ],
+  });
+
+  let trainingDateLabel = intl.formatMessage({
+    defaultMessage: "Training date",
+    id: "yd+VEB",
+    description: "Label for training date meta data",
+  });
+  let trainingDateValue;
+
+  if (trainingOpportunity.trainingStart) {
+    trainingDateValue = formatDate({
+      date: parseDateTimeUtc(trainingOpportunity.trainingStart),
+      formatString: "MMMM d, yyyy",
+      intl,
+    });
+  }
+
+  if (trainingOpportunity.trainingStart && trainingOpportunity.trainingEnd) {
+    trainingDateLabel = intl.formatMessage({
+      defaultMessage: "Training dates",
+      id: "yK0vlP",
+      description: "Label for training dates meta data",
+    });
+
+    trainingDateValue = `${formatDate({
+      date: parseDateTimeUtc(trainingOpportunity.trainingStart),
+      formatString: "MMMM d, yyyy",
+      intl,
+    })} - ${formatDate({
+      date: parseDateTimeUtc(trainingOpportunity.trainingEnd),
+      formatString: "MMMM d, yyyy",
+      intl,
+    })}`;
+  }
+
+  return (
+    <>
+      <SEO
+        title={trainingOpportunity.title?.localized ?? ""}
+        description={intl.formatMessage({
+          defaultMessage:
+            "Learn more about this training opportunity and apply.",
+          id: "7L19hE",
+          description: "Subtitle for training opportunity single page",
+        })}
+      />
+      <Hero
+        title={trainingOpportunity.title?.localized}
+        subtitle={intl.formatMessage({
+          defaultMessage:
+            "Learn more about this training opportunity and apply.",
+          id: "7L19hE",
+          description: "Subtitle for training opportunity single page",
+        })}
+        crumbs={crumbs}
+        buttonLinks={[
+          {
+            icon: ArrowTopRightOnSquareIcon,
+            text: intl.formatMessage({
+              defaultMessage: "Apply now",
+              id: "PH68o/",
+              description:
+                "Label for apply now button in instructor led training",
+            }),
+            url: trainingOpportunity.applicationUrl?.localized ?? "",
+            color: "primary",
+            external: true,
```

Comment:
Adding the "external" icon to both the button icon and the text deviates from the design and seems redundant.  I'd suggest checking in Brooke about ths one.

Reply:
@BrookeCameron and I chatted about the design and it will be changing slightly: the link in the `Hero` will be moved to a `DataRow` at the end of the Course details card with a more standard style and the link at the end of the page will have a secondary colour.